io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Samuel Schlemper -->
+<component type="desktop-application">
+
+<id>io.github.SamuelSchlemperSchlemuel.SingWriter</id>
+<metadata_license>CC0-1.0</metadata_license>
+<project_license>GPL-3.0-or-later</project_license>
+
+<name translate="no">SingWriter</name>
+<summary>Write in SignWriting</summary>
+<summary xml:lang="pt_BR">Escreva em SignWriting</summary>
+
+<developer id="io.github.SamuelSchlemperSchlemuel">
```

Comment:
```suggestion
<developer id="io.github.Samuel_Schlemper_Schlemuel">
```

Reply:
this also apply to the application id, for the same reasons.